package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "template-config",
-  "version": "5.0.19",
+  "version": "5.0.20",
```

Comment:
When things are being removed, do you think it would be better to bump the major version rather than the patch? It probably doesn't matter, just a thought.

Reply:
yes, good point - a minor version bump makes much more sense